docs/ui-configuration.md

```diff
@@ -0,0 +1,25 @@
+## UI Configuration
+
+The UI configuration supports the following params:
+
+- As defined in type [UIOptionsObject](https://github.com/kaltura/playkit-js-ui/blob/master/docs/configuration.md)
+- css - Applying an external style sheet URL is possible as described in [External CSS](#External-CSS)
+
+##
+
+### External CSS
+
+An external style sheet URL can be given in the configuration as follows:
+
+```js
+{
+  ui: {
+    css: 'https://myDomain.com/path/to/css'; // place a valid css file URL String
+  }
+}
+```
+
+The styling will be applied on the entire page.
```

Comment:
What is the reason we mention this?
I think it is obvious that any css definition will be set for entire page as this is just a css file.
Better to add a link to our CSS guide directly so anyone wanting to create the file will know how to set a proper CSS definition: https://github.com/kaltura/playkit-js-ui/blob/master/docs/css-classes-override.md